README.md

```diff
@@ -148,6 +148,22 @@ apt::source { 'puppetlabs':
 },
 ~~~
 
+### Add an Apt source served over HTTPS
+
+~~~puppet
+include apt::transport_https
+
+apt::source { 'nodesource':
+  location => "https://deb.nodesource.com/node_0.10",
+  release  => $::lsbdistcodename,
+  repos    => 'main',
+  key      => {
+    'id'     => '9FD3B784BC1C6FC31A8A0A1C1655A0AB68576280',
+    'source' => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key',
+  }
+}
```

Comment:
Honestly, classes should not be declaring `ca-certificates` themselves at all. That is such a common package it should be left to someone's base layer. Your solution doesn't fix it either since in that case including `apt::transport_https` will still break until every other module stops trying to manage things outside its purview.